src/test/compile-fail/deriving-no-inner-impl-error-message.rs

```diff
@@ -12,8 +12,8 @@ struct NoCloneOrEq;
 
 #[derive(PartialEq)]
 struct E {
-    x: NoCloneOrEq //~ ERROR binary operation `==` cannot be applied to type `NoCloneOrEq`
-         //~^ ERROR binary operation `!=` cannot be applied to type `NoCloneOrEq`
+    x: NoCloneOrEq //~ ERROR binary operation `==` cannot be applied to type `&NoCloneOrEq`
+         //~^ ERROR binary operation `!=` cannot be applied to type `&NoCloneOrEq`
```

Comment:
Yes, but we have `impl<'a, 'b, A, B> PartialEq<&'b B> for &'a A where B: ?Sized, A: PartialEq<B> + ?Sized` so it seems to me it would always be the same impl?